homeassistant/components/plugwise/coordinator.py

```diff
@@ -47,12 +52,16 @@ async def _async_update_data(self) -> PlugwiseData:
         """Fetch data from Plugwise."""
         try:
             data = await self.api.async_update()
-        except XMLDataMissingError as err:
+        except (InvalidXMLError, ResponseError) as err:
             raise UpdateFailed(
-                f"No XML data received for: {self.api.smile_name}"
+                "Invalid XML data, or error indication received for the Plugwise Adam/Smile/Stretch"
             ) from err
-        except PlugwiseException as err:
-            raise UpdateFailed(f"Updated failed for: {self.api.smile_name}") from err
+        except UnsupportedDeviceError as err:
+            raise UpdateFailed(
+                "Unsupported device found, please create an Issue in the Home Assistant Core github"
+            ) from err
+        except ConnectionFailedError as err:
+            raise UpdateFailed from err
```

Comment:
Should this have a message?

Reply:
The coordinator-code already provides a message, at least, it did when I tested this several months ago.
But it looks like the code in `update_coordinator.py` has changed? Related to the boolean `log_failures`?

Reply:
Added as suggested.